public/locales/en/signups.json

```
@@ -0,0 +1,19 @@
+{
+  "attendeeStatus": {
+    "attending": "Attending",
+    "waitlisted": "In wait list"
```

```suggestion
    "waitlisted": "In waiting list"
```
I believe this should be the more common/natural term

---

Changed